### ISSUE TITLE
Exclude paths from vue-remark

### ISSUE THREAD

**Issue by cco3** [gridsome/gridsome#1002]
`source-filesystem` allows specifying multiple paths so that you can list both included paths and excluded paths.

However, vue-remark simply passes in `'**/*.md'`, so none of that flexibility is retained.  It'd be nice to be able to modify that default `path` value.

Example:
```
options: {
  ...
  path: [
    '**/*.md',
    '!**/README.md',
  ],
}
```

**Comment by EldoranDev** [comment]
I created a PR that implents it in the way you suggest.

But while implementing it I was thinking about either renaming it to `ignore` or `exlude` as I think you'll always want to have `**/*.md` inside of your `baseDir`. 

Some optionos on this ? 

My suggested config then would be: 

```js
options: {
  ...
  ignore: [
    '**/README.md',
  ],
}
```

vue-remark then creates a merged array out of `**/*.md` and the ignored paths.

**Comment by cco3** [comment]
I think it makes the most sense just to pass parameters down to source-filesystem.  What if someone wants `**/*.markdown` instead for example?  Exposing the underlying flexibility seems simplest.

**Comment by songololo** [comment]
I'm having a hard time understanding how the `ignore` option works: I seem to be able to be able to exclude specific files, e.g.  this:
```js
ignore: ['**/index.md']
```
will successfully exclude all `index.md` files in my directories, but it is not possible to do the opposite where I ignore everything except a certain file name, i.e. this:
```js
ignore: ['**/*', '!**/index.md']
```
does not work. (I've tried all sorts of permutations, none of which seems to do the trick.)

Looking under the hood, the `ignore` option is concatenated with the default path of `**/*` using a `!` for any additional items, so maybe the double negation is breaking something?
i.e., once processed the above becomes:
```js
paths = ['**/*', '!**/*', '!!**/index.md']
```
Long story short, I'm not sure how to successfully ignore all markdown files except the `index.md` in the root of each of my directories...
